Makefile.COMMON

```diff
@@ -44,12 +44,17 @@ endif
 GO_VERSION ?= 1.4.1
 GOURL      ?= https://golang.org/dl
 GOPKG      ?= go$(GO_VERSION).$(GOOS)-$(GOARCH)$(RELEASE_SUFFIX).tar.gz
-GOROOT     := $(CURDIR)/.build/go
 GOPATH     := $(CURDIR)/.build/gopath
 GOCC       ?= $(GOROOT)/bin/go
 GO         ?= GOROOT=$(GOROOT) GOPATH=$(GOPATH) $(GOCC)
 GOFMT      ?= $(GOROOT)/bin/gofmt
 
+ifeq ($(shell go version | sed 's/.*go\([0-9.]*\).*/\1/'), $(GO_VERSION))
```

Comment:
Following up on https://github.com/prometheus/haproxy_exporter/pull/25/files#r24598112, this will now always print `/bin/sh: 1: go: not found` when running any make target, even `make clean`.

But it seems this gets rid of that error output:

    ifeq ($(shell which go >/dev/null 2>&1 && go version | sed 's/.*go\([0-9.]*\).*/\1/'), $(GO_VERSION))